docs/development/matterbridge.rst

```diff
@@ -1,52 +1,61 @@
-************
-Matterbridge
-************
+******************
+Matterbridge Setup
+******************
 
 ========
 Overview
 ========
 
 We use Matterbridge to connect channels across different messaging platforms
-such as Slack, Mattermost and Gitter. Matterbridge provides binaries for many
-operating systems. These files are intended to be used in combination with
-TOML configuration files::
+such as *Slack*, *Mattermost* and *Gitter*. Matterbridge provides binaries 
+for  many operating systems. The ``matterbridge`` binary is intended to be used
+in combination with a TOML configuration file::
   $ ./matterbridge -conf config-tardis-matterbridge.toml
 
-The TOML file includes all the configuration needed to connect as many
-messaging platforms as you want, including tokens and passwords.
+The TOML file includes all the parameters required to connect as many
+messaging platforms as you want, like *tokens* and *passwords*. Once the
+application is running, messages start be shared between the connected rooms.
 
-Currently, we keep a service running on `OpenSupernova.org`_ server to keep
-running Matterbridge as a daemon. Our configuration files are stored in the
-private repository ``tardis-matterbridge``. These files include our custom
-TOML configuration file and the files needed to set up the Linux service.
+Currently, we keep a service running on `OpenSupernova.org`_ server to 
+run Matterbridge as a daemon. Our configuration files are stored in a private 
+GitHub repository, includin our custom TOML and the files needed to set up the
+Linux service.
 
-Since this server runs Ubuntu 14.04 we use an `Upstart script`_ instead a
-`Systemd service`_ file. But a *Systemd* file is also included in
-``tardis-matterbridge`` repository, just in case.
+Since this server runs Ubuntu 14.04 we use an `Upstart script`_ instead of a
+`Systemd service`_ file, but a Systemd file is also included in our repository.
+A *cron* job restarts the service periodically to prevent disconnections.
 
 
 =====
 Setup
 =====
 
 Use the TOML file in ``tardis-matterbridge`` as an example. You can set up
-as many gateways as you want.
+as many gateways as you want!.
+
 
 -----
 Slack
 -----
 
 Follow the `steps for Slack in the Matterbridge wiki`_ and read carefully
-`this comment`_. Do not add any scopes manually. Copy the token to use later
-in the TOML file.
+`this comment`_ (do not add any scopes manually!). Copy the token to use
+later in the TOML file.
 
 
 ------
 Gitter
 ------
 
 Create a new GitHub dedicated account to use as a bot, refer to `this link`_
-and copy the token.
+and copy the token o use later in the TOML file.
```

Comment:
```suggestion
and copy the token to use later in the TOML file.
```

Just a small typo-- I'm not sure if that's what I should be looking for or not though.

Reply:
Yes, it is. Thanks! :smile: